lib/phoenix/live_dashboard/telemetry_listener.ex

```diff
@@ -29,19 +29,25 @@ defmodule Phoenix.LiveDashboard.TelemetryListener do
     send(parent, {:telemetry, entries})
   end
 
+  def prepare_entry(metric, measurements, metadata) do
+    measurement = extract_measurement(metric, measurements)
+    label = tags_to_label(metric, metadata)
+    {label, measurement}
+  end
```

Comment:
We should tell folks to simply call this function instead of calling both `extract_measurements` and `tags_to_label`. This function can also return `{label, measurements, System.system_time(:microseconds)}`, reducing the amount of knowledge the historical data system has on telemetry. Also note we need to call `keep?` accordingly (per the telemetry guide), so this function may return `nil`, which means there is nothing to be stored.